cypress/pages/elements.page.js

```diff
@@ -12,9 +12,15 @@ export default class ElementsPage {
       .click();
   }
   clickOnTextBox() {
-    cy.get(elementsOnElementsPage.TEXT_BOX)
-      //   .contains("TextBox")
-      .should("be.visible")
-      .click();
+    cy.get(elementsOnElementsPage.TEXT_BOX).should("be.visible").click();
+  }
+  checkIfAllLabelsArePresented() {
+    let allLabels = [
+      cy.get(elementsOnElementsPage.FUL_NAME_LABEL),
+      cy.get(elementsOnElementsPage.EMAIL_LABEL),
+      cy.get(elementsOnElementsPage.ADRESS_LABEL),
+      cy.get(elementsOnElementsPage.PERMANENT_ADRES_LABEL),
+    ];
```

Comment:
Should wait a bit (works with 50). Otherwise, it checks only the last element of the array.